### PR TITLE
Core dump config RHLE

### PR DESCRIPTION
**Pull request by aritosteles**
## Description
Modified instructions for core dump generation in RHLE based OSs.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/development/coredump.rst [hunk 1]
```diff
@@ -8,8 +8,44 @@ Configuring core dump generation
...
+
+#. Edit the Systemd ``/etc/sysctl.d/core.conf`` file. Add the following lines.
+
+   .. code-block:: console
+
+      kernel.core_pattern = /var/lib/coredumps/core-%e-pid%p-time%t
+      kernel.core_uses_pid = 1
+      fs.suid_dumpable = 2
+
+#. Create directory ``/var/lib/coredumps`` and grant it permissions (773).
```
Comment (javimed) [inline source/development/coredump.rst:29]: ` ``773`` ` could use code style font and no need of parenthesis.

File: source/development/coredump.rst [hunk 2]
```diff
@@ -8,8 +8,44 @@ Configuring core dump generation
...
+
+#. Reboot the system
+
+#. After system reboot set the core ulimit to unlimited in your terminal.
+
+   .. code-block:: console
+
+      ulimit -c unlimited
+      sysctl -p
```
Comment (javimed) [inline source/development/coredump.rst:38]: Please add prompt symbols. For example:

```
      # ulimit -c unlimited
      # sysctl -p
```

File: source/development/coredump.rst [hunk 3]
```diff
@@ -8,8 +8,44 @@ Configuring core dump generation
...
+
+#. Restart wazuh agent:
+
+   .. code-block:: console
+
+      ./var/ossec/bin/wazuh-control restart
```
Comment (javimed) [inline source/development/coredump.rst:44]: Copy to clipboard can discard prompt symbols. Please add the prompt symbol for consistency with other code blocks in documentation

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/development/coredump.rst (hunk 1 above)
Comment (aritosteles) [reply source/development/coredump.rst]: Done!

File: source/development/coredump.rst (hunk 2 above)
Comment (aritosteles) [reply source/development/coredump.rst]: Done!

File: source/development/coredump.rst (hunk 3 above)
Comment (aritosteles) [reply source/development/coredump.rst]: Done!

File: source/development/coredump.rst [hunk 4]
```diff
@@ -8,8 +8,44 @@ Configuring core dump generation
...
+
+      kernel.core_pattern = /var/lib/coredumps/core-%e-pid%p-time%t
+      kernel.core_uses_pid = 1
+      fs.suid_dumpable = 2
+
+#. Create directory ``/var/lib/coredumps`` and grant it permissions ``773``.
+
+#. Reboot the system
+
+#. After system reboot set the core ulimit to unlimited in your terminal.
```
Comment (javimed) [inline source/development/coredump.rst:33]:
```suggestion
#. After system reboot set the core ``ulimit`` to ``unlimited`` in your terminal.
```

**Review by javimed — APPROVED** [review]

**Review by ncvicchi — APPROVED** [review]
LGTM!

**Review by javimed — APPROVED** [review]

**Pull request merged**